package.json

```diff
@@ -19,8 +21,8 @@
     "lint"
   ],
   "scripts": {
-    "start": "nodemon src/index.js",
-    "lint": "semistandard src/index.js",
+    "start": "nodemon src/server/index.js",
```

Comment:
I also talked with Reza about this and agreed that separating the server from the client code is a useful step